lib/components/home/artwork_rail.js

```diff
@@ -19,6 +19,7 @@ class ArtworkRail extends React.Component {
 
     this.state = {
       expanded: false,
+      hasAdditionalContent: true,
```

Comment:
This is useful for hiding/showing the View All button and will likely be removed in the future if we pass this value down with Relay.